.kokoro/firebase/codelab-friendlychat-android.cfg

```diff
@@ -4,5 +4,5 @@ build_file: "repository-gardener/.kokoro/build-android.sh"
 
 env_vars: {
     key: "DPEBOT_REPO"
-    value: "firebase/friendlychat-android"
+    value: "codelab-firebase/friendlychat-android"
```

Comment:
I think you mean:
`firebase/codelab-friendlychat-android`

Reply:
Thank you sir!